packages/cosmic-swingset/src/sim-params.js

```diff
@@ -69,13 +69,32 @@ export const defaultPowerFlagFees = [
 ];
 
 export const QueueInbound = 'inbound';
-
 export const defaultInboundQueueMax = 1_000;
-
 export const defaultQueueMax = [
   makeQueueSize(QueueInbound, defaultInboundQueueMax),
 ];
 
+export const vatCleanupDefault = 'default';
+export const defaultVatCleanupDefault = 0;
```

Comment:
As above, let's set the default to 5, so we get limiting right off the bat

Having all the other values at 0 would override that limiting, right? So I think the default budget should either be `{ default: 5 }` or `{ default: 5, kv: 50 }`, or maybe setting everything to 5, or maybe setting everything to 5 but `kv: 50`. But I think the default budget should *not* have anything at 0, because that would permit unlimited cleanup for that category (and thus increase the chances of crashing the kernel), even if a governance thing replaces those 0s for some (but not all) of the values.

Reply:
> But I think the default budget should _not_ have anything at 0, because that would permit unlimited cleanup for that category (and thus increase the chances of crashing the kernel), even if a governance thing replaces those 0s for some (but not all) of the values.

`0` isn't "no limit", it's the lowest possible limit—which I thought was "disallow", but upon closer inspection that turns out not to be the case... [`cleanupAfterTerminatedVat`](https://github.com/Agoric/agoric-sdk/blob/69f8e4b5e04b66df5400716251e66fbce468c86e/packages/SwingSet/src/kernel/state/kernelKeeper.js#L957-L1076) doesn't check remaining budget until _after_ doing some cleanup, which means that `0` is effectively the same as `1`. But at any rate, the current fully-zero defaults in default-params.go and launch-chain.js are maximally conservative, and IIUC, you would prefer the former to instead default to `5` with a KV override of `50`.

Reply:
Ah, ok, I had assumed `0` was magic (it certainly behaved badly at one point during development, before I made vatKeeper more robust), but I think you're right that it's effectively equal to `1`. If that's the maximally-conservative initial setting, I'd rather express as `1` than as `0` to prevent people like me from assuming it's somehow magic :). But yes, I'd prefer the initial values to be `5` and `50` since that what I've measured.